README.md

```diff
@@ -1 +1,235 @@
+values change, calculating the aggregated time series information and storing it in the MongoDB instance.
+This option is called the minimalist option.
+2. A new sink can be enabled in the Cygnus component to calculate and to update the aggregated time series information
+as the entity attribute values change over time. This option is the formal one.
+
+Regarding the aggregated time series information provided by the STH component, there are 4 main concepts which are
+important to know about:
+
+* <b>Range</b>: The period of time about which the aggregated time series information is provided. Possible valid
+ranges values are: year, month, day, hour, minute.
+* <b>Resolution</b> or <b>aggregation period</b>: The time period by which the aggregated time series information is grouped.
+Possible valid resolution values are: month, day, hour, minute and second. For the time being, we only consider the
+following range-resolution pairs: year-month, month-day, day-hour, hour-minute and minute-second.
+* <b>Origin</b>: For certain range-resolution pair, it is the origin of time for which the aggregated time series
+information applies. For example, for a pair hour-minute, a valid origin value could be: ``2015-03-01T13:00:00.000Z```,
+meaning the 13th hour of March, the 3rd, 2015. The origin is stored using UTC time to avoid locale issues.
+* <b>Offset</b>: For certain range-resolution pair, it is the offset from the origin for which the aggregated time series
+information applies. For example, for a pair hour-minute and an origin ```2015-03-01T13:00:00.000Z```, an offset of 10
+refers to the 10th minute of the concrete hour pointed by the origin. In this example, there would be a maximum of 60
+offsets from 0 to 59 corresponding to each one of the 60 minutes within the concrete hour.
+* <b>Samples</b>: For a quadruple range-resolution-origin-offset, it is the number of samples, values, events or notifications available.
+
+The STH component exposes an HTTP ReST API to let external clients query this aggregated time series information. A
+typical URL querying for this information using a GET request is the following:
+
+<pre>http://localhost:8666/STH/v1/contextEntities/type/&lt;quantity&gt;/id/&lt;entityId&gt;/attributes/&lt;attributeId&gt;?aggrMethod=sum&aggrPeriod=second&dateFrom=2015-02-22T00:00:00&dateTo=2015-02-22T23:00:00</pre>
+
+The entries between "<" and ">" in the URL path depend on the concrete case (type of data, entity and attribute) being queried.
+
+The requests can make use the following query parameters:
+
+* <b>aggrMethod</b>: The aggregation method. The STH component supports the following aggregation methods: max (maximum
+value), min (minimum value), sum (sum of all the samples) and sum2 (sum of the square value of all the samples). Combining
+the information provided by these aggregated methods with the number of samples, it is possible to calculate probabilistic
```

Comment:
Yeah, I included it for an easier reading in an IDE :) Do you prefer avoiding this carriage returns? :)